Compiler/src/main/java/com/adguard/compiler/FilterUtils.java

```diff
@@ -100,11 +103,12 @@ public static void loadEnglishFilterForSafari(File destDir) throws IOException {
      * This filters contains some special fix rules for safari content blocker
      *
      * @param destDir Destination directory.
+     * @param filterDownloadUrl
      * @throws IOException
      */
-    public static void loadMobileSafariFilter(File destDir) throws IOException {
+    public static void loadMobileSafariFilter(File destDir, String filterDownloadUrl) throws IOException {

```

Comment:
We don't need this method, safari filter is downloaded as ordinary local filter.